src/ui/locales/ar.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "تبديل الإحالة",
+    "AttributionControl.MapFeedback": "علق على الخريطة",
+    "FullscreenControl.Enter": "أدخل وضع ملء الشاشة",
+    "FullscreenControl.Exit": "الخروج من وضع ملء الشاشة",
+    "GeolocateControl.FindMyLocation": "البحث عن موقعي",
+    "GeolocateControl.LocationNotAvailable": "الموقع غير متوفر",
+    "LogoControl.Title": "Mapbox شعار",
+    "Map.Title": "خريطة",
+    "NavigationControl.ResetBearing": "إعادة تعيين الإتجاة للشمال",
+    "NavigationControl.ZoomIn": "تكبير",
+    "NavigationControl.ZoomOut": "تصغير",
+    "ScaleControl.Feet": "قدم",
+    "ScaleControl.Meters": "م",
+    "ScaleControl.Kilometers": "كم",
+    "ScaleControl.Miles": "ميل",
+    "ScaleControl.NauticalMiles": "ميل بحري",
+    "ScrollZoomBlocker.CtrlMessage": "استخدم مفتاح ctrl + التمرير لتكبير الخريطة",
```

Comment:
@stepankuzmin seems like this one got reversed again 😢. I tried writing the whole line in slack fist. Let me know if it messes up the json format.
```suggestion
,"استخدم مفتاح ctrl + التمرير لتكبير الخريطة" : "ScrollZoomBlocker.CtrlMessage"
```

Reply:
Looks like it probably did. Unsure what to do about this one honestly. I could try to search for what "ctrl" would translate to so that it becomes all arabic letters if that helps?

Reply:
@oielbanna Yes, I think that may work if there is a word for this, thanks

Reply:
```suggestion
    "ScrollZoomBlocker.CtrlMessage": "إستخدم مفتاح التحكم + التمرير لتكبير الخريطة",
```